client/main/tableview.js

```diff
@@ -501,9 +514,12 @@ const TableView = SilkyView.extend({
             }
 
             if (changes.levelsChanged || changes.measureTypeChanged || changes.dataTypeChanged || changes.columnTypeChanged) {
-                $header.attr('data-measuretype', column.measureType);
                 $header.attr('data-columntype', column.columnType);
                 $header.attr('data-datatype', column.dataType);
+                $header.attr('data-measuretype', column.measureType);
+
+                $column.attr('data-columntype', column.columnType);
+                $header.attr('data-datatype', column.dataType);
```

Comment:
$header.attr('data-datatype', column.dataType);
I think this has now been added twice. Here and three lines above